build.env

```diff
@@ -34,6 +34,9 @@ SNAPSHOT_VERSION=v6.1.0
 #TEST_COVERAGE=html
 #GO_COVER_DIR=_output/
 
+# URL for the script to install Helm
+HELM_SCRIPT=https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
```

Comment:
Using script from main branch is risky, possible to use latest released version?

Reply:
This is what is recommended in the documentation. I expect that the previous URL pointed to the same version.

Reply:
Also note that it is only used for installing. The version pointed to by `HELM_VERSION` will be installed and used.

Reply:
I do not think that is needed. Hopefully their main branch is at least as stable as our devel branch.